src/tools.cc

```diff
@@ -1132,8 +1131,8 @@ keepCapabilities(void)
 static void
 restoreCapabilities(bool keep)
 {
-    /* NP: keep these two if-endif separate. Non-Linux work perfectly well without Linux syscap support. */
-#if USE_LIBCAP
+    /* keep these two if-endif separate. Non-Linux work perfectly well without Linux syscap support. */
```

Comment:
If we are polishing this comment: I could not easily tell what exactly should be kept separate. I suggest mentioning specific condition(s) instead. For example:

```suggestion
    // Check HAVE_LIBCAP separately from _SQUID_LINUX_: Some non-Linux cases work without LIBCAP.
```

or

```suggestion
    // Check HAVE_LIBCAP separately from _SQUID_LINUX_: Non-Linux cases may benefit from LIBCAP but work without it.
```

... or whatever gets the reader closer to the actual reasons behind the existing code logic.

Reply:
Fixed in [1f09e63](https://github.com/squid-cache/squid/pull/1718/commits/1f09e63bfe697ac380eb8eeabbb1329080511941)